packages/datetime/src/dateRangePicker.tsx

```diff
@@ -295,6 +301,29 @@ export class DateRangePicker extends AbstractPureComponent2<IDateRangePickerProp
         }
     }
 
+    private isToday = (date: Date) => DateUtils.areSameDay(date, new Date());
```

Comment:
can we make this a new util function in `DateUtils`? that way we can share it with `datePicker.tsx` and we can remove the private class method in both places.